common/persistence/sql/sqlTaskManager.go

```diff
@@ -282,8 +298,14 @@ func (m *sqlTaskManager) ListTaskQueue(request *persistence.ListTaskQueueRequest
 			Name:        lastRow.Name,
 			TaskType:    lastRow.TaskType,
 		})
-	case pageToken.ShardID+1 < m.nShards:
-		nextPageToken, err = gobSerialize(&taskQueuePageToken{ShardID: pageToken.ShardID + 1, TaskType: math.MinInt16})
+	case shardLessThan < math.MaxUint32:
+		var lastShardId uint32
+		if len(rows) == 0 {
+			lastShardId = shardLessThan
```

Comment:
I thought shardLessThan is inclusive.  Should this be +1 so we don't get duplicates?